codemp/client/cl_console.cpp

```diff
@@ -478,19 +479,22 @@ void CL_ConsolePrint( const char *txt) {
 		}
 
 		// count word length
-		for (l=0 ; l< con.linewidth ; l++) {
-			if ( txt[l] <= ' ') {
-				break;
+		if ((l < 0 ? l = 0, true : false) || (l + 1 == con.linewidth)) {
```

Comment:
I understand what it does, but code that takes effort to understand isn't great, especially when it doesn't add any benefit over writing it out normally.